src/main/java/de/rytrox/varo/teams/GameTimeService.java

```diff
@@ -0,0 +1,88 @@
+package de.rytrox.varo.teams;
+
+import de.rytrox.varo.Varo;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerLoginEvent;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalTime;
+
+/**
+ * Service that manages the GameTime based on the internal System-Time
+ *
+ * @author Timeout
+ */
+public class GameTimeService implements Listener {
+
+    private final Varo main;
+
+    private final LocalTime startTime;
+    private final LocalTime endTime;
+
+    private final String kickMessage;
+
+    public GameTimeService(@NotNull Varo main) {
+        this.main = main;
+
+        this.startTime = LocalTime.parse(main.getConfig().getString("teams.start", "16:00"));
+        this.endTime = LocalTime.parse(main.getConfig().getString("teams.end", "22:00"));
+
+        kickMessage = ChatColor.translateAlternateColorCodes('&',
+                "&8[&6Varo&8] &cDie maximale Zeit für diesen Tag ist erreicht. \n" +
+                        String.format("&cDas Spiel wird morgen um %d:%d fortgesetzt", startTime.getHour(), startTime.getMinute()));
+
+        registerEndScheduler();
+    }
+
+    /**
+     * Creates and starts the End-Scheduler.
+     * This scheduler kicks all players from the server that are online at the end-time
+     */
+     public void registerEndScheduler() {
+        Bukkit.getScheduler().runTaskLater(main, () ->
+            Bukkit.getScheduler().runTaskTimer(main, () ->
+                Bukkit.getOnlinePlayers().forEach(player ->
+                    player.kickPlayer(this.kickMessage)
+                )
+            , 0, 24 * 60 * 60 * 20)
```

Comment:
konstante definieren

Reply:
Wieso ne Konstante? Der Compiler übernimmt das doch automatisch

Reply:
Der Compiler ja, aber für jemanden der den Quelltext liest ist es nicht unbedingt direkt ersichtlich. Gabs da von Bukkit nicht sogar was für? Oder war das nur in Bungeecord